mitodl/edx-sysadmin
# Add Git import and Gitlogs code

#### Pre-Flight checklist


#### What are the relevant tickets?
#4  

#### What's this PR do?
Add Git import and logs code (just moving the code from OpenEdx)

#### How should this be manually tested?
Either run through the management command like
`python manage.py lms git_add_course https://github.com/edx/edx4edx_lite.git`
or the below method
1. Just see that import courses are showing in the Git Logs
2. See the logs are also showing
3. I have used that [course](https://github.com/edx/edx4edx_lite.git) if you are willing to use the same course 

#### Where should the reviewer start?
1. Enable Sysadmin dashboard by `ENABLE_SYSADMIN_DASHBOARD` to `True`
2. Add course from old sysadmin through Course tab
<img width="1276" alt="Screenshot 2021-03-24 at 18 53 17" src="https://user-images.githubusercontent.com/4043989/112452307-4f3e3180-8d78-11eb-96d1-033a78489c2c.png">

2. switch back to this sysadmin version after setting `ENABLE_SYSADMIN_DASHBOARD` to `False`

3. And see that the Git Logs tab with logs

#### Any background context you want to provide?
I suppose that Loading new courses from Github should be a part of `Courses` that should have been covered already in [Course Ticket](https://github.com/mitodl/edx-sysadmin/issues/3) that but not yet. So I will be putting up a separate PR for that.

#### Screenshots (if appropriate)
<img width="1440" alt="Screenshot 2021-03-25 at 14 29 36" src="https://user-images.githubusercontent.com/4043989/112452123-18681b80-8d78-11eb-885f-2b9ae92611a1.png">
<img width="1440" alt="Screenshot 2021-03-25 at 14 29 51" src="https://user-images.githubusercontent.com/4043989/112452140-1dc56600-8d78-11eb-8798-d11a982c6bd2.png">
<img width="1440" alt="Screenshot 2021-03-25 at 14 30 06" src="https://user-images.githubusercontent.com/4043989/112452142-1e5dfc80-8d78-11eb-8364-cb6d2b8a854c.png">
<img width="1440" alt="Screenshot 2021-03-25 at 14 30 16" src="https://user-images.githubusercontent.com/4043989/112452146-1f8f2980-8d78-11eb-8d5e-7ba48fc58302.png">
<img width="1440" alt="Screenshot 2021-03-25 at 14 30 23" src="https://user-images.githubusercontent.com/4043989/112452149-20c05680-8d78-11eb-9b4b-ceac8da62b4f.png">
